src/OnboardingSPA/components/Loaders/Step/index.js

```diff
@@ -0,0 +1,17 @@
+import CommonLayout from '../../Layouts/Common';
+import HeadingWithSubHeading from '../../HeadingWithSubHeading';
+import NeedHelpTag from '../../NeedHelpTag';
+
+const StepLoader = ( { title, subtitle } ) => {
+	return (
+		<CommonLayout className="step-loader" isVerticallyCentered>
+			<HeadingWithSubHeading title={ title } subtitle={ subtitle } />
```

Comment:
Just a reminder we need a story for HeadingWithSubHeading to be more appropriately designed for multiline Headings and Subheadings.

<img width="524" alt="Screenshot 2022-09-30 at 10 49 54 AM" src="https://user-images.githubusercontent.com/48691514/193195953-972040bc-4716-40f5-91c5-ee1610b89af2.png">

Reply:
Handled this in the next PR where this state is being used, had to add `line-height: 1`